docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -1236,15 +1236,15 @@ The type environment for the arguments is the same as for the call.
 
 A valid call with results *resolves* to a set of predicates. The ways a call can resolve are as follows:
 
--  If the call has no receiver and the predicate name is a simple identifier, then the predicate is resolved by looking up its name and arity in the visible member-predicate environment of the enclosing class.
+-  If the call has no receiver and the predicate reference is a simple identifier, then the predicate is resolved by looking up its name and arity in the visible predicate environment of the enclosing class.
 
--  If the call has no receiver and the predicate name is a simple identifier, then the predicate is resolved by looking up its name and arity in the visible predicate environment of the enclosing module.
+-  If the call has no receiver and the predicate reference is a simple identifier, then the predicate is resolved by looking up its name and arity in the visible predicate environment of the enclosing module.
 
--  If the call has no receiver and the predicate name is a selection identifier, then the qualifier is resolved as a module (see "`Module resolution <#module-resolution>`__"). The identifier is then resolved in the exported predicate environment of the qualifier module.
+-  If the call has no receiver and the predicate reference is a selection identifier, then the qualifier is resolved as a module (see "`Module resolution <#module-resolution>`__"). The identifier is then resolved in the exported predicate environment of the qualifier module.
 
--  If the type of the receiver is the same as the enclosing class, the predicate is resolved by looking up its name and arity in the visible predicate environment of the class.
+-  If the the call has a receiver and the type of the receiver is the same as the enclosing class, the predicate name is resolved by looking up its name and arity in the visible predicate environment of the class.
 
--  If the type of the receiver is not the same as the enclosing class, the predicate is resolved by looking up its name and arity in the exported predicate environment of the class or domain type.
+-  If the the call has a receiver and the type of the receiver is not the same as the enclosing class, the predicate name is resolved by looking up its name and arity in the exported predicate environment of the class or domain type.
```

Comment:
I think it should still say `...the predicate _ is resolved by looking up its name...`

Reply:
I rephrased them all as ... the call is resolved ... instead, since the table is supposed to describe all the ways calls can resolve.